indent.lisp

```diff
@@ -18,13 +18,29 @@
 
 (defvar *indentation-hints* (make-hash-table :test #'eq))
 
+(defmacro with-symbol (name-symbol-package-module &body body)
+  "Check if SYMBOL exists in PACKAGE when MODULE is loaded and bind it to NAME."
+  (destructuring-bind (name symbol package module)
+      name-symbol-package-module
```

Comment:
Why not just `(defmacro with-symbol ((name symbol package module) &body body)`

Reply:
Oh, I didn't know you could do that.

Reply:
Macro lambda lists are quite a bit more powerful than regular ones! You can destructure recursively like that as much as you want.